specs/swagger.yaml

```diff
@@ -341,14 +351,76 @@ paths:
             $ref: "#/definitions/Peptide"
         '400':
           description: Bad request. The requested parameters are not supported.
-        '401':
-          description: Authorization information is missing or invalid.
+          schema:
+            $ref: "#/definitions/Error"
+        '404':
+          description: No data found for the specified parameters.
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+  /proteins:
+    get:
+      summary: Get a collection of proteins
+      operationId: getProteins
+      description: The protein entreypoint returns a protein statistics across an entire resource.
+      tags:
+      - molecules
+      produces:
+      - "application/json"
+      parameters:
+      - name: pageSize
+        in: query
+        description: How many items to return at one time (default 100, max 100)
+        required: false
+        type: integer
+        default: 100
+      - name: pageNumber
+        in: query
+        description: Current page to be shown paged peptides (default page 1)
```

Comment:
The phrase "paged peptides" in this sentence is grammatically out of place. I suggest simply "Current page of peptide results to be shown (default 1)".